.github/workflows/nix.yaml

```
@@ -90,8 +89,7 @@ jobs:
       - name: Check matrix conditions
         id: check-conditions
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:32: Double quote to prevent globbing and word splitting

(shellcheck)